### PR TITLE
VM: Reactivate more slow tests

### PR DESCRIPTION
**Pull request by holgerd77**
Follow-up on #2937 

I had the impression that the static call tests currently still skipped are also fast enough to be reintegrated.

This is a first test run on all HFs.

WIP, will look into some more additional tests.

### CODE REVIEW

**Review by g11tech — APPROVED** [review]
lgtm

**Review by jochem-brouwer — APPROVED** [review]

**Comment by jochem-brouwer** [comment]
Istanbul and MuirGlacier are failing because the test amount is incorrect

Istanbul: 38340

**Comment by jochem-brouwer** [comment]
I think due to https://github.com/ethereumjs/ethereumjs-monorepo/pull/2971 tests decreased.

Small note: if we update ethereum-tests to a new version, we should always use `test all hardforks` as label (did not notice this at that PR).

The CI should pass now, but will wait for approval until it does.

**Review by jochem-brouwer — APPROVED** [review]

**Pull request merged**
